src/wallet/wallet_rpc_server.cpp

```diff
@@ -1356,6 +1356,17 @@ namespace tools
       return  false;
     }
 
+    std::set<uint32_t> subaddr_indices;
+        if (req.subaddr_indices_all)
+        {
+          for (uint32_t i = 0; i < m_wallet->get_num_subaddresses(req.account_index); ++i)
+            subaddr_indices.insert(i);
+        }
+        else
+        {
+          subaddr_indices= req.subaddr_indices;
```

Comment:
Didnt you forget about something?

Reply:
I reverted fix wallet display issues at the moment